static/src/stylesheets/module/_adslot.scss

```diff
@@ -69,9 +69,17 @@
 }
 
 .sticky-top-banner-ad {
+    transform: translate3d(0, 0, 0);
+    will-change: transform;
```

Comment:
It might be worth adding `will-change` in the JS in the fastdom read directly before you fastdom write the CSS class for animating  - hence giving the browser time to optimise and then [removing it when done](https://dev.opera.com/articles/css-will-change-property/#remove-will-change-after-the-changes-are-done) so the browser can recover resources.